bosh/ruby-app-release/unit-tests/spec/job_spec.rb

```diff
@@ -32,22 +32,22 @@
       expect {template.render("bootstrap" => "config.ru")}.to raise_error 'Wrong bootstrap file provided'
     end
 
-    it 'raises error if control script is malformed' do 
+    it 'raises error if control script is malformed' do
       tmps = template.render("bootstrap" => "app.rb")
       expect(tmps.lines[0]).to include ("#!/bin/bash")
     end
-    
+
     it 'raises error if exec command is malformed' do
       tmps = template.render("bootstrap" => "app.rb")
-
-      exec_line = tmps.each_line do |line|
```

Comment:
This assignment didn't function on my end anymore, it might be that this syntax got outdated. I used the same logic in my update to find the line containing bundle exec